<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tablesData</class>
 <widget class="QDialog" name="tablesData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BD Théâtre (V1)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="tabSpectacles">
      <attribute name="title">
       <string>LesSpectacles</string>
      </attribute>
      <widget class="QTableWidget" name="tableSpectacles">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>11</y>
         <width>1031</width>
         <height>471</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>noSpec</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nomSpec</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prixBaseSpec</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>dateRep</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelSpectacles">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>490</y>
         <width>1031</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabRepresentations">
      <attribute name="title">
       <string>LesRepresentations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableRepresentations">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>dateRep</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>promoRep</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>prixRep</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_representations">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlaces">
      <attribute name="title">
       <string>LesPlaces</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tablePlaces">
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>noPlace</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>noRang</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>noZone</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_places">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabZones">
      <attribute name="title">
       <string>LesZones</string>
      </attribute>
      <widget class="QTableWidget" name="tableZones">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1051</width>
         <height>491</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>38</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>noZone</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>catZone</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelZones">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>499</y>
         <width>1031</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabCategorie">
      <attribute name="title">
       <string>Categorie</string>
      </attribute>
      <widget class="QTableWidget" name="tableCategorie">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>11</y>
         <width>1001</width>
         <height>491</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>catZone</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>tauxZone</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelCategorie">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>500</y>
         <width>1001</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabPersonnes">
      <attribute name="title">
       <string>LesPersonnes</string>
      </attribute>
      <widget class="QTableWidget" name="tablePersonnes">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>11</y>
         <width>1031</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>typePer</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>tauxPer</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelPersonnes">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>490</y>
         <width>1001</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabVentes">
      <attribute name="title">
       <string>LesVentes</string>
      </attribute>
      <widget class="QTableWidget" name="tableVentes">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>991</width>
         <height>461</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>noVente</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prixVente</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelVentes">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>486</y>
         <width>1011</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabDossiers">
      <attribute name="title">
       <string>LesDossiers</string>
      </attribute>
      <widget class="QTableWidget" name="tableDossiers">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>11</y>
         <width>1011</width>
         <height>461</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>noDos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prixTotal</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="labelDossiers">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>486</y>
         <width>991</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>tablesData</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshSpectaclesData()</slot>
  <slot>refreshZonesData()</slot>
  <slot>refreshRepresentationsData()</slot>
 </slots>
</ui>
